tests/lib/rules/prefer-wait-for.test.ts

```diff
@@ -41,6 +41,25 @@ ruleTester.run(RULE_NAME, rule, {
         await testingLibrary.waitFor(() => {}, { timeout: 500 });
       }`,
     },
+    {
+      code: `import { wait } from 'imNoTestingLibrary';
+
+      async () => {
+        await wait();
+      }`,
+    },
+    {
+      code: `import * as foo from 'imNoTestingLibrary';
+
+      async () => {
+        await foo.wait();
+      }`,
+    },
+    {
+      code: `
+      cy.wait();
+      `,
+    },
```

Comment:
I think we should consider testing `require` scenarios as well

Reply:
It would be nice if we can have an additional test for it, indeed.